index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css"> 
+    <title>My Portolio Website</title>
+</head>
+<body>
+    <main class="wrapper">
+        <header class="menu">
+            <div class="welcome-logo">
+                <a href="#">Welcome</a>
+            </div>
+            <nav class="menu-button">
+                <ul>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                </ul>
+            </nav>
+        </header>
+    </main>
+    <section class="hero">
+        <h2 class="hidden">Hero</h2>
+        <section class="headline">
+            <div class="intro primary">
+                <h1> Hey there. I’m Alvin <br>
+                    <span class="green"> I’m a software developer </span></h1>
+            </div>
```

Comment:
- [ ] This heading is almost done, I strongly suggest adding more space between the subtitle and the paragraph:

Template:
![Captura de pantalla 2022-03-18 a la(s) 6 45 37 p  m](https://user-images.githubusercontent.com/83717285/159097814-4d4a7625-c2ae-43db-912e-f97fdca82d61.png)

Your design:
![Captura de pantalla 2022-03-18 a la(s) 6 45 28 p  m](https://user-images.githubusercontent.com/83717285/159097824-5713438b-47ef-48a8-8fb4-090aa5659fe8.png)